python/setup.py

```diff
@@ -96,7 +96,7 @@ def filter_source(filename):
     if "ios/" in filename:
         return False
 
-    if "RegisterPluginsInit_min" in filename:
+    if "addDefaultPluginFactories_min" in filename:
```

Comment:
Didn't you capitalize the file name?
```suggestion
    if "AddDefaultPluginFactories_min" in filename:
```